### ISSUE TITLE
Keep getting scratch buffer...

### ISSUE THREAD

**Issue by ssssssssk** [rougier/emacs-splash#2]
Hi,

Love how the splash screen works on Nano Emacs, was thrilled to find a similar stand alone version on your GitHub. I'm able to call the splash buffer into existence, but not able to get it to appear before anything else. Instead I get dumped in the scratch buffer without seeing the splash screen. 

I don't have an initial buffer set. 

Any suggestions?

**Comment by rougier** [comment]
How do you start emacs (from command line?) and what is your system?

**Comment by ssssssssk** [comment]
I usually us spotlight (on Mac) or dmenu/rofi (on Linux). 

When I wrote you last night I had already put the package in my load path, I had told my init to require the package, and then I even set the initial buffer to load the splash screen.

I was able to make it work by changing line 160 from:
(defun mac-animation-fade-out (duration &rest args)
to 
(defun mac-animation-fade-out (duration &rest _args)

After that it worked fine.

I was even able to simply modify the behavior to go to a scratch buffer instead of to the about screen and a few other things that suit my workflow better. 

Now that it's working for me this is a very nice alternative to the Emacs Dashboard package, and with all the screen sharing going on with remote working I'm so glad to not have emacs open to my agenda all the time, the fade out adds a touch of class and it's easy to work with as a package (since I don't know much about lisp). 

Thanks for all your great design work on emacs by the way.
